### PR TITLE
Release v4.0.43.2

### PR DESCRIPTION
**Pull request by hy-ca**

### CODE REVIEW

File: site/pages/docs/versions/dwnld-en.hbs [hunk 1]
```diff
@@ -25,12 +25,12 @@
 		<tbody>
 			<tr>
 				<td>Web Experience Toolkit (WET)</td>
-				<td>v4.0.43.1 (STR)</td>
+				<td>v4.0.43.2</td>
 				<td class="text-center"><a class="btn btn-default" href="https://github.com/wet-boew/wet-boew/releases/latest">Download<span class="wb-inv"> - v4.0.43 (STR) - Web Experience Toolkit (WET) theme</span></a></td>
```
Comment (GormFrank) [inline site/pages/docs/versions/dwnld-en.hbs:29]:
```suggestion
				<td class="text-center"><a class="btn btn-default" href="https://github.com/wet-boew/wet-boew/releases/latest">Download<span class="wb-inv"> - v4.0.43.2 - Web Experience Toolkit (WET) theme</span></a></td>
```

File: site/pages/docs/versions/dwnld-en.hbs [hunk 2]
```diff
@@ -25,12 +25,12 @@
...
 			</tr>
 			<tr>
 				<td>Canada.ca (GCWeb)</td>
-				<td>v9.4.1 (STR)</td>
+				<td>v9.5.0</td>
 				<td class="text-center"><a class="btn btn-default" href="https://github.com/wet-boew/GCWeb/releases/latest">Download<span class="wb-inv"> - v9.4.0 (STR) - Canada.ca theme</span></a></td>
```
Comment (GormFrank) [inline site/pages/docs/versions/dwnld-en.hbs:34]:
```suggestion
				<td class="text-center"><a class="btn btn-default" href="https://github.com/wet-boew/GCWeb/releases/latest">Download<span class="wb-inv"> - v9.5.0 - Canada.ca theme</span></a></td>
```

**Review by GormFrank — CHANGES_REQUESTED** [review]
You just forgot two spots

**Review by GormFrank — CHANGES_REQUESTED** [review]
Don't forget to update line 13 in the package.json file, since WET-BOEW will be updated.

File: site/pages/docs/versions/dwnld-en.hbs (hunk 2 above)
Comment (hy-ca) [reply site/pages/docs/versions/dwnld-en.hbs]: @GormFrank Completed required changes.

**Review by GormFrank — APPROVED** [review]

**Pull request merged**